.gitmodules

```diff
@@ -8,7 +8,7 @@
 	branch = dune
 [submodule "coq-master"]
 	path = coq-master
-	url = https://github.com/coq/coq.git
+	url = https://github.com/ejgallego/coq.git
 	branch = master
```

Comment:
The submodule patch doesn't exist in master, you probably want to push your changes to a branch and point there.

Reply:
I see the submodule commit just fine, as does the CI